STEER/AOD/AliAODInputHandler.cxx

```diff
@@ -106,7 +106,8 @@ Bool_t AliAODInputHandler::Init(TTree* tree, Option_t* opt)
 
     fTree = tree;
     if (!fTree) return kFALSE;
-    GetEntries();
+    //GetEntries();
```

Comment:
Why having to comment this out? It is currently used to check the friend open status stored in the chain by TChain::GetEntries(). Disabling it will make grid jobs with bad files crash only when reaching those, instead of crashing in the beginning of the job.

Reply:
In TChain::GetEntries(), the last tree gets loaded `const_cast<TChain*>(this)->LoadTree(TTree::kMaxEntries-1);`. Shouldn't in the first step the first tree be loaded (thus the LoadTree(0) I added)?

Reply:
No, LoadTree(TTree::kMaxEntries-1) forces the chain to discover the number of entries in every file from the chain. This is needed for the chain to setup its internals, and now we also use the status stored per file entry in the chain to detect if the file could be opened or not. If we comment out GetEntries, this is not checked and the job will crash during the event loop (I tested this). Normally LoadTree() gets called automatically by GetEntry() (called by the selector in the event loop), so I don't see why adding LoadTree(0) is needed